### PR TITLE
Coffee shop detail icon

### PR DESCRIPTION
**Pull request by uwgnol1612**

### CODE REVIEW

**Review by hegelocampus — CHANGES_REQUESTED** [review]
Those accessibility issues that were marked by accesslint bot need to be addressed, **employers heavily value attention to accessibility issues**

File: client/src/components/main/coffee_shop/CoffeeShop.js [hunk 1]
```diff
@@ -69,10 +69,22 @@ export default () => {
         <div className="coffee-shop-details">
           <h1>Coffee Shop details for {name}</h1>
           <ul>
-            <li>Founded in {founded}</li>
-            <li>{type}</li>
-            <li><a href={ url }>{url}</a></li>
-            <li>Barista Satisfaction: {baristaSatisfaction}</li>
+            <li>
+              <img src="https://we-camp-seeds.s3.us-east-2.amazonaws.com/Coffee+bar+year.png" />
```
[the remark this answers is not in the document]
Comment (uwgnol1612) [reply client/src/components/main/coffee_shop/CoffeeShop.js]: Fixed

File: client/src/components/main/coffee_shop/CoffeeShop.js [hunk 2]
```diff
@@ -69,10 +69,22 @@ export default () => {
...
+              Founded in {founded}
+            </li>
+            <li>
+              <img src="https://we-camp-seeds.s3.us-east-2.amazonaws.com/Coffee+bar+type.png" />
```
[the remark this answers is not in the document]
Comment (uwgnol1612) [reply client/src/components/main/coffee_shop/CoffeeShop.js]: Fixed

File: client/src/components/main/coffee_shop/CoffeeShop.js [hunk 3]
```diff
@@ -69,10 +69,22 @@ export default () => {
...
+              {type}
+            </li>
+            <li>
+              <img src="https://we-camp-seeds.s3.us-east-2.amazonaws.com/Coffee+website.png" />
```
[the remark this answers is not in the document]
Comment (uwgnol1612) [reply client/src/components/main/coffee_shop/CoffeeShop.js]: Fixed

File: client/src/components/main/coffee_shop/CoffeeShop.js [hunk 4]
```diff
@@ -69,10 +69,22 @@ export default () => {
...
+              <a href={url}>{url}</a>
+            </li>
+            <li>
+              <img src="https://we-camp-seeds.s3.us-east-2.amazonaws.com/Coffee+satisfaction.png" />
```
[the remark this answers is not in the document]
Comment (uwgnol1612) [reply client/src/components/main/coffee_shop/CoffeeShop.js]: Fixed

**Review by hegelocampus — APPROVED** [review]
Great! Thank you!

**Pull request merged**